packages/core/src/services/latitudeTools/webExtract/index.ts

```diff
@@ -77,16 +78,9 @@ export default {
   method: webExtract,
   definition: (context: TelemetryContext) => ({
     description: 'Given a URL, returns the contents of the page.',
-    parameters: {
-      type: 'object',
-      properties: {
-        url: {
-          type: 'string',
-        },
-      },
-      required: ['url'],
-      additionalProperties: false,
-    },
+    inputSchema: z.object({
```

Comment:
Now Vercel wants `inputSchema` instead of `parameters` and a zod object

Reply:
have we changed agent/integration/client tools?